src/components/views/right_panel/UserInfo.tsx

```diff
@@ -292,13 +292,17 @@ function DevicesSection({ devices, userId, loading }: {devices: IDevice[], userI
     let expandButton;
     if (expandSectionDevices.length) {
         if (isExpanded) {
-            expandButton = (<AccessibleButton className="mx_UserInfo_expand mx_linkButton"
+            expandButton = (<AccessibleButton
+                kind="link"
+                className="mx_UserInfo_expand"
                 onClick={() => setExpanded(false)}
             >
                 <div>{ expandHideCaption }</div>
             </AccessibleButton>);
         } else {
-            expandButton = (<AccessibleButton className="mx_UserInfo_expand mx_linkButton"
+            expandButton = (<AccessibleButton
+                kind="link"
```

Comment:
why are these two `link` and the others `link_inline`?

Reply:
I'll get ready screenshots to explain why (I cannot remember quite well)

Reply:
Here, `link_inline` breaks the layout because these two `AccessibleButtons` contain an E2E shield icon, so `link` has been applied. But setting `link` to everything instead of `link_inline` looks consistent (they are not inline elements), so I am going to change the others into `link`.